source/includes/gcp/_k8_releases.md

```diff
@@ -374,3 +398,40 @@ Mandatory | &nbsp;
 Optional | &nbsp;
 ------- | -----------
 `values` <br/>*string* | YAML structured text that will overwrite the default values for the upgrade/installation of the chart.
+
+<!-- UNINSTALL RELEASE -->
+### Uninstall a release
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/k8s/anenvironment/releases/pspensieri/aerospike-1579797954&operation=uninstall"
+   -d "request_body"
+
+# Request body example
+```
```

Comment:
I feel like the ``` should be before the `# Request body example` for this

Reply:
I guess it must not be there, there is no example

Reply:
I agree, I left the `# Request body example` inside the curl block to stay consistent with the rest of our docs. The right hand side becomes messy when we have both request body response examples.